common/changes/@bentley/ui-components/presentation-highlighted-matches_2020-11-11-14-35.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@bentley/ui-components",
+      "comment": "Updated FilteringInput and filterers, implemented some other features enabling highlighting of the records",
```

Comment:
Similar to the other changelog message, this needs to tell what changed

Reply:
Changed the message

Reply:
I would change it to:
```
Updated [[FilteringInput]] component. Component's `filteringInProgress` prop was deprecated and new `status` property was introduced. This allows rendering the component in any state, including "filtering finished", without having to cycle through other states first.
```

Reply:
Updated changeFile